container-storage-setup.sh

```diff
@@ -1589,14 +1609,21 @@ activate_extra_lv_fs() {
 # activate command processing start
 run_command_activate() {
   local metafile_path="$_CONFIG_DIR/$_CONFIG_NAME/$_METAFILE_NAME"
+  local status_file="$_CONFIG_DIR/$_CONFIG_NAME/$_STATUSFILE_NAME"
+  local curr_status
 
   [ ! -d "$_CONFIG_DIR/$_CONFIG_NAME" ] && Fatal "Storage configuration $_CONFIG_NAME does not exist"
 
   [ ! -e "$metafile_path" ] && Fatal "Storage configuration $_CONFIG_NAME metadata does not exist"
   source "$metafile_path"
 
+  curr_status=`cat $status_file`
```

Comment:
good point. I should handle exit code of cat. So that if status file does not exist, we should error out early.